english/eating-walls/eating-marshmallows.md

```diff
@@ -5,7 +5,7 @@ Sitting down next to the partially eaten marshmallow wall, your stomach begins t
 and babble. Hoping to walk off the uneasy feeling, you stand up. Upon standing up, your stomach
 quiets and you entertain how to spend the rest of the day.
 
-But wait!  Just when you when you were about to start contemplating, you suddenly feel torn between 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.